appservice/src/intTest/groovy/nu/yona/server/EditGoalsTest.groovy

```diff
@@ -51,10 +51,12 @@ class EditGoalsTest extends AbstractAppServiceIntegrationTest
 		gamblingGoals.size() == 1
 		gamblingGoals[0]."@type" == "BudgetGoal"
 		!gamblingGoals[0]._links.edit //mandatory goal
+		gamblingGoals[0].creationTime =~ /\d{4}\-\d{2}\-\d{2}T\d{2}\:\d{2}\:\d{2}\.\d{3}\+\d{2}\:\d{2}/
```

Comment:
Can't you add an assert that the creation time is equal to the time the user was created?